src/scanner.h

```diff
@@ -185,7 +187,7 @@ typedef UInt            TypSymbolSet;
                     |S_PLUS|S_MINUS|S_NOT|S_LPAREN)
 
 #define STATBEGIN  (S_IDENT|S_UNBIND|S_IF|S_FOR|S_WHILE|S_REPEAT \
-                    |S_BREAK|S_RETURN|S_HELP|S_QUIT)
+                    |S_BREAK|S_RETURN|S_HELP|S_QUIT|S_PRAGMA)
```

Comment:
This is not actually needed, as S_HELP now implies S_PRAGMA

Reply:
Fixed.